bundles/org.eclipse.packagedrone.utils.rpm/src/org/eclipse/packagedrone/utils/rpm/build/RpmBuilder.java

```diff
@@ -139,6 +142,89 @@ public String toString ()
         }
     }
 
+    public static class Feature
+    {
+        private String name;
+
+        private String evr;
+
+        private EnumSet<RpmDependencyFlags> flags;
+
+        private String description;
+
+        public Feature ()
+        {
+
+        }
+
+        public Feature ( String name, String evr, EnumSet<RpmDependencyFlags> flags, String description )
```

Comment:
It might be great to have another constructor like:

~~~java
public Feature ( String name, String version, String description, RpmDependencyFlags ... flags ) {
  this(name, version, description, EnumSet.of(flags));
}
~~~

Reply:
Sure. I see that this class matches the existing `Dependency` class a lot (which has a constructor like this). But, now I wonder if this class should be combined somehow with that class.

Reply:
Sure, maybe you can make a common base class. Or re-use the dependency class.